examples/src/main/java/io/github/tcdl/msb/examples/FacetsAggregator.java

```diff
@@ -86,7 +86,7 @@ public static void main(String[] args) throws ScriptException, FileNotFoundExcep
                     responder.send(responsePayload);
                 });
 
-                requester.publish(request);
+                requester.publish(request, responder.getOriginalMessage().getCorrelationId());
```

Comment:
what means inside? we pass this to Requester constructor. now we pass this to publish method.